src/components/FormModal/FormModal.css

```diff
@@ -1,14 +1,15 @@
 .FormModal {
   position: absolute;
   width: 100vw;
-  height: 100vh;
+  height: 600px;
```

Comment:
I think it’s better to use em or rem for resizing, unless you want this height fixed.

Reply:
i want it fixed :) But thx for the tip